### PR TITLE
ADT-139 - Change modelId delimiter

### PR DESCRIPTION
**Pull request by alistairknight**
# **What this PR does / why we need it**:
* The delimiter used for model_id should be replaced with a pipe | instead of a colon :. This is to avoid conflicts as some LLM deployment services also use colons in their model ids
* [Jira ticket](https://eaflood.atlassian.net/jira/software/projects/ADT/boards/1565/backlog?assignee=712020%3Aefe0c96c-4f9d-4a0d-97e6-f8711455e97d&selectedIssue=ADT-139)

# **Special notes for your reviewer**
* This PR is linked to [https://github.com/DEFRA/coreai-mcu-prompts/pull/6](https://github.com/DEFRA/coreai-mcu-prompts/pull/6)
* RowKey values in local table storage **models** should have colons (:) replaced with pipes (|)
* New data in **models** table should use pipe for 
* PartitionKey values in local table storage **prompts** should have colons (:) replaced with pipes (|)
* New data in **prompts** table should use pipe character, e.g. `POST` to `http://localhost:3006/prompts`:
`{
    "project": "mcu",
    "name": "Correspondence",
    "modelId": "azure|openai|gpt-53-turbo",
    "prompt": "[INST] \nYou are an expert in writing responses to correspondence from the general public. You are assisting a Defra employee in writing a response to the document contained in [DOCUMENT] below. \n \nYou should only provide a response based on the knowledge contained in [CONTEXT]. You should not infer any additional information. \n \nIf you are unable to find any relevant information in [CONTEXT], you should respond with a message stating that you are \"unable to provide a response\". \n \nYour response should always be in the format of an email or a letter. You should always sign the response with \"Yours sincerely, [NAME] [ROLE]\". \n \nYour response should always be addressed to the member of the public. \n[EXAMPLE] \nYou receive a email that a member of parliament (Joe Bloggs MP) has forwarded to Defra from their constituent (Alice Simpson). \n[CORRECT_RESPONSE] \nIn this case you should tailor your response to Alice Simpson and not Joe Bloggs MP. \n[/CORRECT_RESPONSE] \n[/EXAMPLE] \n[/INST] \n \n[DOCUMENT] \n{document} \n[/DOCUMENT] \n \n[CONTEXT] \n{context} \n[/CONTEXT]",
    "type": "correspondence"
}`

# Testing
*Any relevant testing information and pipeline runs.*

# Checklist (please delete before completing or setting auto-complete)
- [ ] Story Work items associated (not Tasks)
- [ ] Successful testing run(s) link provided
- [ ] Title pattern should be `{work item number}: {title}`
- [ ] Description covers all the changes in the PR
- [ ] This PR contains documentation
- [ ] This PR contains tests
- [ ] Version number in **package.json** has been updated

# **How does this PR make you feel**:
gif

### CODE REVIEW

**Review by sfitz42 — APPROVED** [review]
LGTM

File: app/services/ai/clients/index.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ const clients = {
 }
 
 const getClient = (modelId) => {
-  const [vendor, service, model] = modelId.split(':')
+  const [vendor, service, model] = modelId.split('|')
 
   const client = clients[`${vendor}:${service}`]
```
Comment (sfitz42) [inline app/services/ai/clients/index.js]: This needs updating to `|`

**Review by sfitz42 — DISMISSED** [review]

File: app/services/ai/clients/index.js (hunk 1 above)
Comment (alistairknight) [reply app/services/ai/clients/index.js]: I've changed the : to | in client/clients

**Review by sebhoron — APPROVED** [review]
LGTM

**Pull request merged**